system-test/errors-transport.ts

```diff
@@ -56,6 +60,12 @@ export class ErrorsApiTransport extends common.Service {
     });
   }
 
+  async request(options: common.DecorateRequestOptions) {
+    return new Promise<ApiResponse>((resolve, reject) => {
+      super.request(options, (err, _, res) => err ? reject(err) : resolve(res));
```

Comment:
While neat, this was hard to read as a one liner.  Can you use more ... punctuation?

Reply:
Can you elaborate? `gts` is fine with this style.
NB. this copied from https://github.com/googleapis/nodejs-logging-winston/pull/254.

Reply:
Changed.